code/src/components/Header.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./header.css";
+
+export const Header = () => {
+  return <header className="main-heading">
+    New albums and singles
+      </header>
+};
+
+
```

Comment:
Use parentheses when returning several lines:
```
export const Header = () => {
  return (
    <header className="main-heading">
      New albums and singles
    </header>
  )
};

```
and one newline is enough at the end